### PR TITLE
IBX-150: Schedule block snapshots

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-150
| Versions      | 2.5+

### CODE REVIEW

File: docs/guide/config_repository.md [hunk 1]
```diff
@@ -196,3 +196,14 @@ ezplatform:
                 content: '/1/18/'
                 media: '/1/57/'
 ```
+
+## Schedule block snapshots
```
Comment (mnocon) [inline docs/guide/config_repository.md:200]: Nitpick: the block is called `Content Scheduler` 😛

Comment (DominikaK) [inline docs/guide/config_repository.md:200]:
```suggestion
## Content Scheduler snapshots
```

File: docs/guide/config_repository.md [hunk 2]
```diff
@@ -196,3 +196,14 @@ ezplatform:
...
+
+Schedule block snapshots speed up the rendering of Schedule blocks and reduce the place used in the database.
```
Comment (DominikaK) [inline docs/guide/config_repository.md:202]:
```suggestion
Content Scheduler snapshots speed up the rendering of Content Scheduler blocks and reduce the place used in the database.
```

File: docs/guide/config_repository.md [hunk 3]
```diff
@@ -196,3 +196,14 @@ ezplatform:
...
+By default, five snapshots are stored, but you can modify this number with the following configuration,
+depending on the complexity of the Schedule blocks:
```
Comment (DominikaK) [inline docs/guide/config_repository.md:204]:
```suggestion
depending on the complexity of the Content Scheduler blocks:
```

**Review by webhdx — APPROVED** [review]

File: docs/guide/config_repository.md [hunk 4]
```diff
@@ -196,3 +196,14 @@ ezplatform:
...
+## Content Scheduler snapshots
+
+Content Scheduler snapshots speed up the rendering of Content Scheduler blocks and reduce the place used in the database.
+By default, five snapshots are stored, but you can modify this number with the following configuration,
+depending on the complexity of the Content Scheduler blocks:
+
+``` yaml
+parameters:
+    ezplatform.fieldtype.ezlandingpage.block.schedule.snapshots.amount: 10
+```
```
Comment (juskora) [inline docs/guide/config_repository.md:209]: Maybe: reduce the space used (...)?

**Review by juskora — APPROVED** [review]

File: docs/guide/config_repository.md [hunk 5]
```diff
@@ -196,3 +196,14 @@ ezplatform:
                 content: '/1/18/'
                 media: '/1/57/'
 ```
+
+## Content Scheduler snapshots
+
+Content Scheduler snapshots speed up the rendering of Content Scheduler blocks and reduce the place used in the database.
```
Comment (DominikaK) [inline docs/guide/config_repository.md:202]:
```suggestion
Content Scheduler snapshots speed up the rendering of Content Scheduler blocks and reduce the space used in the database.
```

**Pull request merged**